components/job-orchestration/job_orchestration/scheduler/search/search_scheduler.py

```diff
@@ -68,36 +68,57 @@ async def release_reducer_for_job(job: SearchJob):
         await job.reducer_handler_msg_queues.put_to_handler(msg)
 
 
-def fetch_new_search_jobs(db_cursor) -> list:
-    db_cursor.execute(
-        f"""
-        SELECT {SEARCH_JOBS_TABLE_NAME}.id as job_id,
-        {SEARCH_JOBS_TABLE_NAME}.search_config
-        FROM {SEARCH_JOBS_TABLE_NAME}
-        WHERE {SEARCH_JOBS_TABLE_NAME}.status={SearchJobStatus.PENDING}
-        """
-    )
-    return db_cursor.fetchall()
+@exception_default_value(default=[])
+def fetch_new_search_jobs(db_conn) -> list:
+    """
+    Fetches and returns new PENDING jobs from the database. If an exception occurs while interacting
+    with the database this function will return an empty list.
```

Comment:
```suggestion
    Fetches search jobs with status=PENDING from the database.
    :param db_conn:
    :return: The pending search jobs on success. An empty list if an exception occurs while
    interacting with the database.
```

We use reStructuredText for our Python docstrings.